src/libraries/Common/src/System/Buffers/ArrayBufferWriter.cs

```diff
@@ -81,15 +81,41 @@ public ArrayBufferWriter(int initialCapacity)
         /// Clears the data written to the underlying buffer.
         /// </summary>
         /// <remarks>
-        /// You must clear the <see cref="ArrayBufferWriter{T}"/> before trying to re-use it.
+        /// You must reset or clear the <see cref="ArrayBufferWriter{T}"/> before trying to re-use it.
         /// </remarks>
+        /// <remarks>
+        /// Using <seealso cref="ResetWrittenCount"/> method is more performance effective.
+        /// </remarks>
+        /// <remarks>
+        /// If you clear the buffer data by <see cref="Clear"/> method, <see cref="GetMemory"/> method will return clean <see cref="Memory{T}"/>.
+        /// </remarks>
```

Comment:
I was unsure whether VS supports multiple `<remarks>` sections for a single method, so I wrote a simple app and tested it with VS 2022:


```cs
internal class IDETest
{
    /// <remarks>First remark.</remarks>
    /// <remarks>Second remark.</remarks>
    /// <remarks>Third remark.</remarks>
    public void MultipleRemarks() { }

    /// <remarks>
    /// <para>First remark.</para>
    /// <para>Second remark.</para>
    /// <para>Third remark.</para>
    /// </remarks>
    public void MultipleParams() { }
}
```

![image](https://github.com/dotnet/runtime/assets/6011991/15e2567c-0351-4a27-b3ba-e079c01b793f)

![image](https://github.com/dotnet/runtime/assets/6011991/a69e8fac-5870-40a0-9a94-b64eeabe1480)

As you can see, it does not. Please use multiple `<param><params>` in a single `<remarks></remarks>` block.